init.js

```diff
@@ -49,6 +62,10 @@ module.exports = {
   // Derived fields are asynchronous functions that are given the previous user
   // input data of the form: `function (data, cb)`. They callback with:
   // `(err, value)`.
+  //
+  // **Note**: We have implied derived values of:
+  // - `{{npmignore}}` -> `.npmignore`
+  // - `{{gitignore}}` -> `.gitignore`
```

Comment:
See this full discussion https://github.com/FormidableLabs/builder-init/pull/15/files#diff-04c6e90faac2675aa89e2176d2eec7d8R224